src/phases/tpch/normalized/Q1.yml

```diff
@@ -0,0 +1,55 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 1 against the normalized schema. Using an 'executionStats' explain causes each command to run its execution plan until no
+  documents remain, which ensures that the query executes in its entirety.
+
+batchSize: &batchSize 101  # The default batch size.
+query1Delta: &query1Delta -90
+
+TPCHNormalizedQuery1:
+  Repeat: 1
+  Database: tpch
+  Operations:
+  - OperationMetricsName: Query1
+    OperationName: RunCommand
+    OperationCommand:
+      explain:
+        aggregate: lineitem
+        pipeline:
+          [
+            {$match: {$expr: {$lte: ["$l_shipdate", {$dateAdd: {startDate: {$dateFromString: {dateString: "1998-12-01"}}, unit: "day", amount: {^Parameter: {Name: "Query1Delta", Default: *query1Delta}}}}]}}},
```

Comment:
These parameters are good, but I think it'll be easy for the fact that the query is parameterized on these values to be lost in the length of the query. Really a flaw in our parameterization syntax.

Would it make sense to move them to the top and use anchors to place them in the query? Like [here](https://github.com/mongodb/genny/blob/69ee0b0391fb35b46b6fbb8d8fae1928157c7169/src/phases/scale/LargeScalePhases.yml#L42).